cmd/api/src/auth/permission.go

```diff
@@ -95,5 +98,7 @@ func Permissions() PermissionSet {
 
 		SavedQueriesRead:  model.NewPermission("saved_queries", "Read"),
 		SavedQueriesWrite: model.NewPermission("saved_queries", "Write"),
+
+		WipeDB: model.NewPermission("wipedb", "wipedb"),
```

Comment:
Style nit here but it seems like we follow a patter of lowercase for authority and then uppercase for name and i'm not sure if we wanted to make the authority generalized such as:
`db` and then for name `Wipe` perhaps?

Reply:
I agree that the naming here could be improved